pkg/acceptance/compose/gss/psql/Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build the test binary in a multistage build.
-FROM golang:1.13 AS builder
+# Note here we are using master's gss_compose tests to do this.
+# As master is on go 1.15, we have to use go 1.15 too.
+FROM golang:1.15 AS builder
```

Comment:
Does this mean that this will build the test and cockroachdb 20.2 with go 1.15?

Reply:
no, just the test binary with go1.15.

Reply:
(this is not different from before this PR; just that we're using the same go version as master)